### DISCUSSION TITLE
Make it possible to pull docker images anonymously from GitHub package registry

### DISCUSSION THREAD

**Discussion by weeco** [community/community#26025] [Packages]
<p>Hello,</p>
<p>as of now it is <strong>not </strong> possible to pull docker images anonymously from the GitHub package registry even though they are absolutely public. This is the killer argument which makes me not going to use the package registry as I want to make sure all users can anonymously pull docker images, e. g. to:</p>
<ul>
<li>
<p>Run services in Kubernetes, directly pulling from GitHub. This would require the annoying setup of image pull secrets and therefore people would likely workaround this by mirroring images to their own registry of choice</p>
</li>
<li>
<p>Run docker image locally. Most users and devs are probably not yet authenticated against the GH docker registry and therefore they can not pull images from it, until they figured out how to do so</p>
</li>
</ul>
<p><strong>TL;DR: Please make it possible to anonymously pull docker images from the docker registry for public repositories.</strong></p>
<p><strong>Note:</strong> I’ve seen a strong desire for this here as well, but it hasn’t get any attention from GitHub staff (maybe because of the lacking usecases?): <a href="https://github.community/t/docker-pull-from-public-github-package-registry-fail-with-no-basic-auth-credentials-error/16358">https://github.community/t5/GitHub-Actions/docker-pull-from-public-GitHub-Package-Registry-fail-with-quot/td-p/32782</a></p>

**Comment by ethomson** [comment, marked as answer]
<p>We appreciate the feedback.  We realize this is a popular feature request.  We’ve passed this along to the GitHub Package Registry team.</p>

**Comment by MaxHorstmann** [comment]
<p>Any update here? This is a real blocker and inconsistent with the general concept of public repos &amp; packages.</p>

**Comment by ben-z** [comment]
<p>Anonymous downloads from container registry was released yesterday: <a href="https://github.blog/2020-09-01-introducing-github-container-registry/" rel="nofollow noopener">https://github.blog/2020-09-01-introducing-github-container-registry/</a></p>
<p>Can’t find where to configure it though :thinking:</p>

**Reply by vanniszsu** [reply]
in the package setting, Danger zone, change visibility

![image](https://github.com/user-attachments/assets/e6a8c50f-0a02-4d21-a04a-0fa3fe363c16)
